app/services/vote_service.rb

```diff
@@ -2,25 +2,43 @@ class VoteService
   def initialize(answer:, auth_token:)
     @answer = answer
     @auth_token = auth_token
+    @change_hash = { positive: 0, negative: 0 }
```

Comment:
Using instance variables, especially when sending them out directly is not a good practice. For example, simple typo like that:
```ruby
notifier.notify(debate, @chenge_hash)
```
would send a `nil` to the external world.

How about
```ruby
def change_hash
  @change_hash ||= { positive: 0, negative: 0 }
end
```
and then using just `#change_hash` method. Also, it's a good opportunity to replace `@auth_token` with `auth_token` by adding private `attr_reader`.